src/main/java/com/nike/cerberus/operation/rds/CreateDatabaseOperation.java

```diff
@@ -99,13 +102,13 @@ public void run(CreateDatabaseCommand command) {
 
     @Override
     public boolean isRunnable(CreateDatabaseCommand command) {
-        try {
-            cloudFormationService.getStackId(configStore.getPrimaryRegion(), Stack.SECURITY_GROUPS.getFullName(environmentName));
-        } catch (IllegalArgumentException iae) {
+        boolean isRunnable = true;
+
+        if (!cloudFormationService.isStackPresent(configStore.getPrimaryRegion(), Stack.SECURITY_GROUPS.getFullName(environmentName))) {
+            isRunnable = false;
             logger.error("The security group stack must exist to create the the data base stack!");
-            return false;
         }
 
-        return !cloudFormationService.isStackPresent(configStore.getPrimaryRegion(), Stack.DATABASE.getFullName(environmentName));
+        return isRunnable;
```

Comment:
Shouldn't the old line be kept?

How about either !isPresent or restoring from snapshot?

Reply:
You are correct, I removed this when I was experimenting with allowing multiple db stacks.